e2e/schemas/petStore.yaml

```diff
@@ -733,6 +733,14 @@ components:
         name:
           type: string
           example: doggie
+        log:
+          type: string
+          maxLength: 42
+          minLength: 2
+          pattern: >-
+            ^[A-Za-z0-9()\[\]'"][-A-Za-z0-9_.
```

Comment:
Are new lines ignored or could issues arise?

Reply:
@meister03 So long it's proper YAML or JSON it will be converted and validated correctly.